nebula.proto

```diff
@@ -26,6 +25,7 @@ message NebulaMetaDetails {
   uint32 VpnIp = 1;
   repeated Ip4AndPort Ip4AndPorts = 2;
   repeated Ip6AndPort Ip6AndPorts = 4;
+  repeated uint32 RelayVpnIp = 6;
```

Comment:
Why was field index `5` skipped in here? We should either fix this to be `5` or document why it was skipped.

Reply:
thx - 5 was used for CIDR advertisements, which I removed for the new config scheme. Can move RelayVpnIp down a notch :)